src/index.html

```diff
@@ -11,7 +11,181 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action=""
+      method="post"
+      onsubmit="onSubmit()"
+    >
+      <fieldset
+        class="forFieldset"
+      >
+        <legend>Personal information</legend>
+        <label
+          for="surname"
+          class="label"
+        >Surname:
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+            required
+          >
+        </label>
+        <label for="name" class="label">Name:
+          <input
+            type="text"
+            id="name"
+            name="name"
+            required
+          >
+        </label>
+        <label for="age" class="label">How old are You?
+          <input
+            type="number"
+            class="age"
+            id="age"
+            name="age"
+            value="12"
+            min="12"
+            max="100"
+          >
+        </label>
+        <label for="birthday" class="label">Full date of birth:
+          <input
+            type="date"
+            id="birthday"
+            name="birthDay"
+          >
+        </label>
+        <label for="term" class="label">I accept the term of the agreement
+          <input
+            type="checkbox"
+            id="term"
+            name="term"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset
+        class="forFieldset"
+      >
```

Comment:
```suggestion
      <fieldset class="forFieldset">
```
Теги с одним атрибутом можно записывать вот так